storage/src/mmr/mem.rs

```diff
@@ -60,14 +60,16 @@
 //!   Hash(18, element_18)                                      // third peak
 //! )
 //! ```
-use crate::mmr::{Hash, Hasher, Proof};
+use crate::mmr::{MmrHasher, Proof};
```

Comment:
In other packages, we'd still call this `Hasher` but just rename or use the prefix path for the other hasher.

So:
```
use crate::mmr::{Hasher, Proof};
use commonware_cryptography::{Digest, Hasher as CHasher};
```

Or 

```
use crate::mmr::{Hasher as MmrHasher, Proof};
use commonware_cryptography::{Digest, Hasher};
```

Reply:
done (picked the first suggestion above)